fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/RemoteScanPartitionPruneRule.java

```diff
@@ -96,6 +96,7 @@ private void initPartitionInfo(LogicalScanOperator operator, OptimizerContext co
                                    Map<ColumnRefOperator, Set<Long>> columnToNullPartitions)
             throws AnalysisException {
         Table table = operator.getTable();
+        operator.getScanOperatorPredicates().clear();
```

Comment:
why add this logic? add some comment pls.

Reply:
done